### PR TITLE
Updated issue templates

### PR DESCRIPTION
**Pull request by wknapik**
Everything can be adjusted, feel free to push to this PR branch, or request changes in comments

This can be tested at https://github.com/brave-experiments/wknapik-test-02/issues/new/choose

image
---
image
---
image
---
image
---
image
---
image

### CODE REVIEW

**Review by mihaiplesa — APPROVED** [review]

**Review by bsclifton — APPROVED** [review]
Looks great! Tried it out and it's very nice

**Review by mihaiplesa — APPROVED** [review]

**Comment by iccub** [comment]
Would be cool to have a `Feature Request` template. One text field and dropdown with a platform list ios,andro,desktop

File: .github/ISSUE_TEMPLATE/02_desktop_and_android.yml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    validations:
+      required: true
+  - type: textarea
+    id: device
+    attributes:
+      label: Device
+      description: Device information
+      value: |-
+        - Install type (ARM, x86): 
+        - Device type (Phone, Tablet, Phablet):
```
[the remark this answers is not in the document]
Comment (SergeyZhukovsky) [reply .github/ISSUE_TEMPLATE/02_desktop_and_android.yml:75]: yeah, just device type would be enough, on most models you can find what CPU do they use

**Comment by TEMP-ad** [comment]
hey @wknapik wouldn't be good to add a specific `Adblocking` button as part of the templates as well? so people might post adblocking related issues in https://community.brave.com/c/support-and-troubleshooting/ad-blocking/ rather than GitHub?
That's where you see FanboyNZ checking adblocking issues mostly, and also other people try to help there, like Yuki2718. I used to do that too, but I am not in the forum anymore so can't help people there with custom rules and solutions.

Technically any filter list related issues should be in `https://github.com/brave/adblock-lists/issues`, but seems like community was made the better place for that.

The thing is that Brave depends mostly on other lists, so the issues that might happen because of Brave specific/made rules are minimal as well, but people might report them to brave anyway, when 3rd party default lists should be reported to the respective filter list repository/group that is causing the issue like Easylist or uAssets, or could be just a rule that will be updated/added by those groups later and Brave doesn't need to do anything and just _wait for it_ (Like youtube). Also issues might be a custom rule/list that people added or accidentally created themselves.

If anything, there is also Brave's `Menu` -> `Help` -> `Report a broken site` feature, which seems the better official/browser way to report adblocking and shield issues in general, but seems like Community with adblocking category is better (?).

**Review by khaldamyn472-rgb — COMMENTED** [review]
.

**Pull request merged**